README.md

```diff
@@ -70,15 +84,21 @@ Stop Zendro service.
 
-4. start all service by executing **`zendro start`**. Default database would be a local Postgres database. Its configuration is in this file: `./graphql-server/config/data_models_storage_config.json`. If user would like to add other storage types, it is necessary to edit this file. Meanwhile, if you would like to use SPA with production mode, please add `-p` option.
+4. start all service by executing **`zendro start`**. Default database would be a local Postgres database. Its configuration is in this file: `./graphql-server/config/data_models_storage_config.json`. If user would like to add other storage types, it is necessary to edit this file. Meanwhile, if you would like to use GQS and SPA with production mode, please add `-p` option.
    
-5. stop all running service by executing **`zendro stop`**. Besides, if you would like to stop SPA with production mode, please add `-p` option.
+5. stop all running service by executing **`zendro stop`**. Besides, if you would like to stop GQS and SPA with production mode, please add `-p` option.
+
+6. If you don't have local database, you can play with Zendro by dockerizing example Zendro App. The command would be **`zendro dockerize -u`**. Moreover, if you would like to use GQS and SPA with production mode, please execute **`zendro dockerize -u -p`**. Besides, the default username is `admin@zen.dro` and the corresponding password is `admin`. 
 
-6. If you don't have local database, you can play with Zendro by dockerizing example Zendro App. The command would be **`zendro dockerize -u`**. Moreover, if you would like to use SPA with production mode, please execute **`zendro dockerize -u -p`**. Besides, the default username is `admin@zen.dro` and the corresponding password is `admin`. 
+7. When you want to stop docker service, press CTRL+C once, then execute **`zendro dockerize -d`**. In addition, if your GQS and SPA is in production mode, please execute **`zendro dockerize -d -p`**. 
 
-7. When you want to stop docker service, press CTRL+C once, then execute **`zendro dockerize -d`**. In addition, if your SPA is in production mode, please execute **`zendro dockerize -d -p`**. 
+### Example for Migrations
+If a user has new data model definitions, it is convinient to use Zendro CLI for dealing with migrations. And the following procedure shows how to generate, perform or drop migrations:
+1. in `graphql-server` folder, execute `zendro migration:generate -f <data_model_definitions>`, then migrations would be generated automatically in `/graphql-server/migrations` folder. By default, every migration file would have two functions, namely `up` and `down` function. In specifically, in `up` function a table or indices would be created and in `down` function the existing table or indices would be deleted. Besides, user can customize their migration functions.
```

Comment:
>in `graphql-server` folder, execute `zendro migration:generate -f <data_model_definitions>`, then migrations would be generated automatically in `/graphql-server/migrations` folder
in `graphql-server` folder, execute `zendro migration:generate -f <data_model_definitions>`. The migrations are automatically generated in the `/graphql-server/migrations` folder.